src/components/buttons/IconicLink.jsx

```diff
@@ -36,15 +36,17 @@ IconicLink.propTypes = {
   icon: PropTypes.string,
   iconColor: PropTypes.string,
   caption: PropTypes.string,
-  buttonClassName: PropTypes.string,
+  orangeButton: PropTypes.bool,
```

Comment:
I don't think we want this to be a prop type because it's not very reusable... why not keep the original `buttonClassName` and pass it either 'button-primary-orange' or 'button-primary-white' ?